packages/repository-json-schema/src/__tests__/integration/build-schema.integration.ts

```diff
@@ -641,5 +680,55 @@ describe('build-schema', () => {
         },
       });
     });
+    it('does not pollute the JSON schema options', () => {
+      @model()
+      class Category {
+        @property()
+        name: string;
+      }
+
+      const JSON_SCHEMA_OPTIONS = {};
+      // tslint:disable-next-line:no-unused
```

Comment:
Question, why do you need the `const categorySchema =` part instead of just calling `getJsonSchema(Category, JSON_SCHEMA_OPTIONS);`?

Reply:
Good catch!